docs/intro/tutorial08.txt

```diff
@@ -48,12 +48,16 @@ The steps are not duplicated in this tutorial, because as a third-party
 package, it may change separately to Django's schedule.
 
 Once installed, you should be able to see the DjDT "handle" on the right side
-of the browser window when you refresh the polls application. Click it to open
+of the browser window when you refresh the admin of your project. Click it to open
 the debug toolbar and use the tools in each panel. See the `panels
 documentation page
 <https://django-debug-toolbar.readthedocs.io/en/latest/panels.html>`__ for more
 information on what the panels show.
 
+To see the toolbar in the polls application in addition to the admin, make sure 
+all your templates are wrapped in complete HTML and their content is 
+inside the ``<body></body>`` tags.
```

Comment:
This is a pretty long sentence and complete HTML is something that probably needs further explanation for beginners.

```suggestion
To see the toolbar in the polls application make sure all your templates
are using `complete HTML documents <https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Getting_started#anatomy_of_an_html_document>`_.
The toolbar will only render in responses that contain the ``<body></body>`` tags.
```

Note: I didn't cover the line-lengths / formatting well.

Reply:
Good one, applied in https://github.com/django/django/pull/18285/commits/4e774ca18c9ae93eece57907c85567c14454f7bb